blocks/shared-styles/_children/headings/context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const LevelContext = React.createContext(1);
```

Comment:
heheh this is kind of funny how simple it is. for a split second, I didn't get it. then h1 hierarchy heh.